README.md

```diff
@@ -85,6 +85,11 @@ $ sudo rpm -Uvh gdrcopy-<version>.<platform>.rpm
 $ sudo rpm -Uvh gdrcopy-devel-<version>.<platform>.rpm
 ```
 
+Non-DKMS kmod package can be generated by adding `-m` option to `build-rpm-packages.sh`.
+Unlike DKMS kmod, this package is specific to the NVIDIA driver version and the
```

Comment:
"unlike the dkms package, a kmod package contains a prebuilt kernel module which is specific ..."